packages/web-react/scripts/prepareDist.js

```diff
@@ -37,19 +36,20 @@ delete packageJson.engines;
 // on-going package development (e.g. running tests, supporting npm link, etc.).
 // When publishing from "dist" however, we need to update the package.json
 // to point to the files within the same directory.
-const distPackageJson =
-  JSON.stringify(
-    packageJson,
-    (_key, value) => {
-      if (typeof value === 'string' && value.startsWith('./dist/')) {
-        const parts = value.split('/');
-        parts.splice(1, 1); // remove dist
-        return parts.join('/');
-      }
-      return value;
-    },
-    2,
-  ) + '\n';
+// const distPackageJson = `${JSON.stringify(
```

Comment:
Let's maybe remove the lines instead of commenting out? Or is the code going to be used soon?

Reply:
Removed.